coala_quickstart/green_mode/filename_operations.py

```diff
@@ -0,0 +1,136 @@
+import os
+from copy import deepcopy
+
+from coala_quickstart.green_mode.green_mode import (
+    append_to_contents)
+
+
+class Node:
+    def __init__(self, character, parent=None):
+        self.count = 1
+        self.character = character
+        self.children = {}
+        self.parent = parent
+
+    def insert(self, string, idx):
+        if idx >= len(string):
+            return
+        code = ord(string[idx])  # ASCII code
+        ch = string[idx]
+        if ch in self.children:
+            self.children[ch].count += 1
+        else:
+            self.children[ch] = Node(string[idx], self)
+        self.children[ch].insert(string, idx+1)
+
+
+class Trie:
+    """
+    Creates a Trie data structure for storing names of files.
+    """
+
+    def __init__(self):
+        self.root = Node('')
+
+    def insert(self, string):
+        self.root.insert(string, 0)
+
+    # Just a wrapper function.
+    def getPrefixes(self, min_length, min_files):
```

Comment:
Any reason to use `mixedCase` instead of `lower_case`?

Reply:
no I basically copied the high level code from some tutorial and then made the changes. I will fix this 😅